src/ansiblelint/prerun.py

```diff
@@ -158,23 +158,47 @@ def prepare_environment() -> None:
     _prepare_ansible_paths()
 
 
+def _get_galaxy_role_ns(galaxy_infos: Dict[str, Any]) -> str:
+    """Compute role namespace from meta/main.yml, including trailing dot."""
+    role_namespace = galaxy_infos.get('namespace', None)
+    if not role_namespace:
+        role_namespace = galaxy_infos.get('author', None)
+    # if there's a space in the name space, it's likely author name
+    # and not the galaxy login, so act as if there was no namespace
+    if re.match(r"^\w+ \w+", role_namespace):
+        role_namespace = ""
```

Comment:
This is also going to raise an error: running a match on a None string. Maybe use a default of empty string would be better for all these fields.

Reply:
ok. Fixed things to use and empty string and made these two functions to always return a possibly empty string. Will push an update soon